src/software/jetson_nano/thunderloop_main.cpp

```diff
@@ -5,8 +5,23 @@
 #include "software/logger/logger.h"
 #include "software/world/robot_state.h"
 
+// clang-format off
+std::string BANNER =
```

Comment:
shouldn't this be a shared constant with full system?

Reply:
it should probably say THUNDERLOOP (updated to say that instead)